lib/src/rules/pub/no_http_urls.dart

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
```

Comment:
nit: "2018" --> "2012"

Reply:
even 2021 :)

Reply:
:-) Yes, that's what I was trying to type.